package-lock.json

```diff
@@ -4142,8 +4142,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
```

Comment:
@antoinerg which version of `npm` are you using?

Your PR doesn't change any of our dependencies, so it shouldn't change the `package-lock`. Can you revert these changes?

Reply:
I now updated to npm `6.8.0`, ran `npm i ` and the `package-lock.json` has now zero changes! Sorry about that... I need a safety check on my dev machine.

Done f628b549433df3f86244125a2e66929781929f69